stable/kiam/values.yaml

```diff
@@ -137,7 +137,7 @@ server:
 
   image:
     repository: quay.io/uswitch/kiam
-    tag: v3.3
+    tag: v3.4
```

Comment:
can you update the readme too - so that it matches? TIA

Reply:
Done